connectors/helidon-connector/src/main/java17/org/glassfish/jersey/helidon/connector/HelidonVersionChecker.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2020, 2025 Oracle and/or its affiliates. All rights reserved.
```

Comment:
If we are precise about the CP years here and in the ProxyBuilder there is also a redundant year. Other new files look fine.